frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -199,21 +200,23 @@ class PopupCreateFile extends React.Component {
 
     Promise.all(state.fileToUploadList.map(async (file) =>
       this.buildUploadedFileResponse(file)
-    )).then(uploadedFileResponseList => {
-      this.handleAllFileUploadEnd(uploadedFileResponseList.map((uploadedFileResponse) => this.handleFileUploadEnd(uploadedFileResponse.xhr, uploadedFileResponse.file)))
+    )).then(async uploadedFileResponseList => {
+      const fileUploadList = await Promise.all(uploadedFileResponseList.map(async (uploadedFileResponse) => this.handleFileUploadEnd(uploadedFileResponse.xhr, uploadedFileResponse.file)))
```

Comment:
is async necessary here ?
Also, reformat code, this line is too long

Reply:
No you are right, it is useless.